### ISSUE TITLE
Angular home.component.html reference re server-side prerendering no longer applies

### ISSUE THREAD

**Issue by DavidChennells** [aspnet/templating#470]
Minor issue here:  On the home page of the angular template (home.component.html), we're still saying, 

> If you want to enable server-side prerendering, see the steps in Startup.cs

However, the inline guidance referred to in Startup.cs was removed in commit 22da82f (Jan 4) by @SteveSandersonMS

**Comment by DamianEdwards** [comment]
Moving this out to RTM and assigning to @SteveSandersonMS

**Comment by muratg** [comment]
Is this in yet? Can this be closed?

**Comment by SteveSandersonMS** [comment]
@muratg No, the PR is still marked as pending shiproom approval: https://github.com/aspnet/templating/pull/496

It looks like we have a miscommunication, because this issue is marked as SHP:Approved but I was watching the PR for approval.

I'm happy to still merge it into `release/2.1` if you think that's desirable, but maybe at this stage it's now better not to do so in 2.1.

**Comment by muratg** [comment]
@SteveSandersonMS We should probably consider this for the next patch. @mkArtakMSFT?

**Comment by mkArtakMSFT** [comment]
Luckily this is minor. The PRs should not be used to track the Shiproom status. Rather the issues should.

**Comment by mkArtakMSFT** [comment]
So this was actually approved. Anyway, I'm moving it to 2.1.1 milestone now.

**Comment by SteveSandersonMS** [comment]
@mkArtakMSFT My apologies. I was not familiar with that aspect of the process.

So I can merge the PR into the correct place, will 2.1.1 be branched from `dev` or from `release/2.1`?

**Comment by mkArtakMSFT** [comment]
@SteveSandersonMS, we'll reuse the same release/2.1 branch for it, after the RTM. Feel free to make the fix in `dev` so only the porting work will be left for this. I'll let you know when this can be merged.

**Comment by SteveSandersonMS** [comment]
This was auto-closed by the PR #496 that I coincidentally merged to `dev` 1 minute before @mkArtakMSFT's comment above.

Reopened so we can track getting this into the 2.1 patch.
